docs/user/getting_started.rst

```diff
@@ -25,17 +25,18 @@ An example configuration can be found in the Gluon repository at *docs/site-exam
 Dependencies
 ------------
 To build Gluon, several packages need to be installed on the system. On a
-freshly installed Debian Stretch system the following packages are required:
+freshly installed Debian Bullseye system the following packages are required:
 
 * `git` (to get Gluon and other dependencies)
-* `subversion`
```

Comment:
If subversion is removed here, it should also be dropped in `install-depenencies.sh` and the `Dockerfile`.

Also, these changes aren't mentioned in the commit message.

Reply:
fixed, thanks!